mayastor/src/replica.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! Replica is a logical data volume exported over nvmf (in SPDK terminology
 //! an lvol). Here we define methods for easy management of replicas.
-
+#![allow(dead_code)]
```

Comment:
is it that the whole file is not used or just part of it?

Reply:
Once this PR lands, it should be removed.